### PR TITLE
Remove indexer management GraphQL API

### PR DESCRIPTION
**Pull request by Jannis**
This API is not supposed to be exposed by the indexer service. It is something that only indexer agent runs to give indexer CLI and other tools the ability to configure the indexer agent behavior.

### CODE REVIEW

**Comment by Jannis** [comment]
@aasseman @hopeyen Fixed up the linter error, so this should be passing all checks now.

File: service/src/server/mod.rs [hunk 1]
```diff
@@ -92,12 +88,6 @@ pub async fn create_server(
             get(routes::deployment::deployment_health
                 .layer(AddExtensionLayer::new(slow_ratelimiter()))),
         )
-        .route(
```
Comment (hopeyen) [inline service/src/server/mod.rs:95]: Should `indexer-service` not serve cost models at all?

**Comment by hopeyen** [comment]
`indexer-service` doesn't serve cost models at all? Does the gateway request cost_models from this API?

File: service/src/server/routes/cost.rs [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+        &self,
+        ctx: &Context<'_>,
+        deployment: String,
+    ) -> Result<Option<CostModel>, IndexerError> {
+        let pool = &ctx.data_unchecked::<ServerOptions>().indexer_management_db;
+        indexer_management::cost_model(pool, &deployment).await
+    }
+}
 
 pub(crate) async fn graphql_handler(
```
Comment (hopeyen) [inline service/src/server/routes/cost.rs:42]: In the spirit of remove indexer management graphQL API, perhaps we remove `graphql_handler` and use a different handler for the `cost` requests?

Comment (Jannis) [reply service/src/server/routes/cost.rs:42]: This is the public cost model API that indexer-service exposes as well — and that _is_ a GraphQL API. So this is correct.

**Review by hopeyen — APPROVED** [review]

**Pull request merged**